experimental/kernels/kernels_c.cpp

```diff
@@ -0,0 +1,525 @@
+#include "gpu.h"
+#include <array>
+#include <cstdio>
+#include <future>
+
+#include "kernels.h"
+#include "kernels_c.h"
```

Comment:
It's going to be confusing to have kernels.h kernels_c.h and kernels_c.cpp.

I think this library is substantial enough perhaps we should graduate from inline quoted strings for the shader code (which is more for small scale standalone examples) to a .wgsl file for each kernel, at which point i think we can basically delete kernels.h and rename kernels_c.h -> kernels.h and kernels_c.cpp -> kernels.cpp.

any reasons that wouldn't work?

Reply:
It would be better to rename kernels_c.h and kernels_c.cpp.
For now, kernels.h includes general-purpose kernels, and kernels_c.h and kernels_c.cpp are positioned as code dedicated to testing llm.c.

Reply:
I renamed these files and moved them to the unittest_llmc directory.